iphone/cli/commands/_build.js

```diff
@@ -1136,6 +1137,7 @@ iOSBuilder.prototype.configOptionPPuuid = function configOptionPPuuid(order) {
 				relistOnError: true,
 				complete: true,
 				suggest: false,
+				autoSelectOne: true,
```

Comment:
Is this flag already implemented or where will this be handled?

Reply:
Yeah, `autoSelectOne` has been a property for "select" fields for years.

Reply:
Ah sweet, thanks for the clarification!